src/client/lib/downloadInstaller.js

```diff
@@ -42,24 +45,23 @@ Make sure the files are available on their respective locations when distributin
 `,
   )
 
-  const installerDownloads = installersData.map(({ branch, manifestName, os })=> {
-    return createInstaller(branch, manifestName, os)
+  const installerDownloads = installersData.map(({ type, env, os })=> {
+    return createInstaller(type, env, os)
   });
 
   return Promise.all(installerDownloads)
 }
 
-// File name + github branch for each json manifest
 const INSTALLERS_TO_CREATE = [
-  { manifestName: 'dev', branch: 'master', os: 'win' },
-  { manifestName: 'dev', branch: 'master', os: 'osx' },
-  { manifestName: 'uat', branch: 'master', os: 'win' },
-  { manifestName: 'uat', branch: 'master', os: 'osx' },
-  { manifestName: 'demo', branch: 'master', os: 'win' },
-  { manifestName: 'demo', branch: 'master', os: 'osx' },
-  { manifestName: 'launcher-dev', branch: 'master', os: 'win' },
-  { manifestName: 'launcher-uat', branch: 'master', os: 'win' },
-  { manifestName: 'launcher-demo', branch: 'master', os: 'win' }
+  { type: 'app', env: 'dev', os: 'win' },
+  { type: 'app', env: 'dev', os: 'osx' },
+  { type: 'app', env: 'uat', os: 'win' },
+  { type: 'app', env: 'uat', os: 'osx' },
+  { type: 'app', env: 'demo', os: 'win' },
+  { type: 'app', env: 'demo', os: 'osx' },
+  { type: 'launcher', env: 'dev', os: 'win' },
+  { type: 'launcher', env: 'uat', os: 'win' },
+  { type: 'launcher', env: 'demo', os: 'win' }
```

Comment:
Once we have made this change, my plan is that we don't actually build installers or check them into source control.

We can use direct links like this: `https://install.openfin.co/download/?os=win&config=https%3A%2F%2Fweb-demo.adaptivecluster.com%2Fopenfin%2Fapp.json&fileName=ReactiveTraderCloud-Demo&unzipped=true`

Or even `fins://web-demo.adaptivecluster.com/openfin/app.json`, which works if the OpenFin runtime is already installed.

Reply:
Nice. I did notice we're committing the installers to git and was going to say, this change should make that redundant. I didn't want to delete them just yet those until this was merged.